src/Client/ClientRegistry.lua

```diff
@@ -56,6 +56,16 @@ function ClientRegistry:LoadCommand(CommandData: Types.NexusAdminCommandData): (
     elseif ExistingCommand then
         CmdrCommandData.ClientRun = ExistingCommand.ClientRun
     end
+
+    -- Force arguments lost during invocation to be included on the client. Extremely shallow.
+	if ExistingCommand then
+		for index,arg in ipairs(ExistingCommand.Args) do
+			if typeof(CmdrCommandData.Args[index]) ~= typeof(arg) then
+				table.insert(CmdrCommandData.Args,index,arg)
```

Comment:
Formatting issues:
- PascalCase is used in the project instead of camelCase.
- `ipairs` isn't used in any other part of the project. While it is more performant, it isn't enough for this application ro demand it.
- `i` is used for indexing by number in other parts of the project.
- Full names are preferred, such as `Argument` over `arg`.
- While this file isn't consistent, new changes are meant to have a space after a comma.

Requested lines:
```lua
for i, Argument in ExistingCommand.Args do
    if typeof(CmdrCommandData.Args[i]) ~= typeof(Argument ) then
        table.insert(CmdrCommandData.Args, i, Argument )
```